### PR TITLE
Renamed example to examples.

### PR DESCRIPTION
**Pull request by JulianFeinauer**
Added Python Quickstart Example.
Added Python Example to the getting-started page.

Signed-off-by: Julian Feinauer <j.feinauer@pragmaticminds.de>

### CODE REVIEW

File: examples/quickstart-python/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    See the NOTICE file(s) distributed with this work for additional
+    information regarding copyright ownership.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0
+
+    SPDX-License-Identifier: EPL-2.0
+ -->
+# Eclispe Hono Quickstart tutorial
```
Comment (sophokles73) [inline examples/quickstart-python/README.md:13]: Eclipse

File: examples/quickstart-python/pom.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    <artifactId>hono-quickstart-python</artifactId>
+    <version>1.3.0-SNAPSHOT</version>
+
+
+    <name>Hono Quickstart Tutorial in Python</name>
+    <description>This is a python example. See the Readme for more information.</description>
+    <url>https://www.eclipse.org/hono</url>
+
+</project>
```
Comment (sophokles73) [inline examples/quickstart-python/pom.xml:33]: missing end-of-line

File: examples/quickstart-python/README.md (hunk 1 above)
Comment (sophokles73) [inline examples/quickstart-python/README.md:13]: Eclipse

File: examples/quickstart-python/src/main/python/quickstart.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    def on_link_error(self, event):
+        print("Link Error")
+
+    def on_message(self, event):
+        print("Got a message:")
+        print(event.message.body)
+
+
+# Prepare the container
+uri = f'amqp://{amqpNetworkIp}:15672/telemetry'
```
Comment (sophokles73) [inline examples/quickstart-python/src/main/python/quickstart.py:94]: I do not think that you need to append `/telemetry` here ..

File: site/homepage/content/getting-started/index.md [hunk 4]
```diff
@@ -35,6 +35,25 @@ The Hono command line client is used in this guide for simulating an application
 The client is available from [Hono's download page]({{< relref "downloads" >}}).
 Note that running the command line client requires a Java 11 runtime environment being installed locally.
 
+#### Very quick Quickstart
+
+In the following all necessary commands and operations will be explained on how to register a tenant and a device via the HTTP Api.
```
Comment (sophokles73) [inline site/homepage/content/getting-started/index.md:40]: ... the HTTP API.

File: site/homepage/content/getting-started/index.md [hunk 5]
```diff
@@ -35,6 +35,25 @@ The Hono command line client is used in this guide for simulating an application
...
+Then, it will be shown how messages can be sent via the HTTP and MQTT Endpoint.
```
Comment (sophokles73) [inline site/homepage/content/getting-started/index.md:41]: ... via the HTTP and MQTT protocol adapters.

File: examples/quickstart-python/README.md (hunk 1 above)
Comment (JulianFeinauer) [reply examples/quickstart-python/README.md]: Fixed.

File: examples/quickstart-python/pom.xml (hunk 2 above)
Comment (JulianFeinauer) [reply examples/quickstart-python/pom.xml]: Fixed.

File: examples/quickstart-python/README.md (hunk 1 above)
Comment (JulianFeinauer) [reply examples/quickstart-python/README.md]: Fixed.

File: site/homepage/content/getting-started/index.md (hunk 5 above)
Comment (JulianFeinauer) [reply site/homepage/content/getting-started/index.md]: Changed.

File: examples/quickstart-python/src/main/python/quickstart.py (hunk 3 above)
Comment (JulianFeinauer) [reply examples/quickstart-python/src/main/python/quickstart.py]: Indeed, it works without and I changed it accordingly.

File: site/homepage/content/getting-started/index.md (hunk 4 above)
Comment (JulianFeinauer) [reply site/homepage/content/getting-started/index.md]: Changed.

**Comment by JulianFeinauer** [comment]
@sophokles73 sorry for the typos. I changed all points accordingly and squashed it to one commit.

**Comment by JulianFeinauer** [comment]
@sophokles73 thanks for the hint! I replaced all mentions and also fixed some (older) outdated links, so I hope the whole site structure is now fine with new repo example layout.

**Comment by JulianFeinauer** [comment]
Just had a look into Travis and it seems that its not related to my changes but rather an issue with docker / redhats repo?

**Comment by sophokles73** [comment]
@dejanb @ctron anything to add?

**Comment by sophokles73** [comment]
@JulianFeinauer can you rebase and resolve the conflicts?
@ctron do you believe we need Python related works-with CQs for this?

**Comment by sophokles73** [comment]
@JulianFeinauer sorry for the delay with this CQ. However, we would like to merge it now so it would be great if you could rebase and remove the conflicts one final time. Thanks a lot for your patience.

**Comment by ctron** [comment]
> @ctron do you believe we need Python related works-with CQs for this?

No, I don't think so.

**Pull request merged**